application/core/SurveyCommonAction.php

```diff
@@ -483,7 +483,7 @@ protected function showadminmenu($aData)
             $aData['dataForConfigMenu']['userscount'] = User::model()->count();
 
             //Check if have a comfortUpdate key
-            if (getGlobalSetting('emailsmtpdebug') != '') {
+            if (getGlobalSetting('update_key') != '') {
```

Comment:
Better to use `App()->getConfig('update_key');`

I really think this function was deprecated ?

https://github.com/LimeSurvey/LimeSurvey/blob/34bf855a7e1f288cb45186a548a7182811e1e0f2/application/helpers/globalsettings_helper.php#L30